common/djangoapps/course_modes/tests/test_views.py

```diff
@@ -35,6 +34,12 @@
 
 from ..views import VALUE_PROP_TRACK_SELECTION_FLAG
 
+# Name of the class to mock for Content Type Gating.
+GATING_CLASS_NAME = 'openedx.features.content_type_gating.models.ContentTypeGatingConfig.enabled_for_enrollment'
```

Comment:
Would it be more appropriate to use the path of the class name and not the .method and then where it's mocked used `GATING_CLASS_NAME.enabled_for_enrollment`?

Reply:
I will change it to `GATING_CLASS_METHOD`.